osbs-client.spec

```diff
@@ -83,12 +83,6 @@ mv %{buildroot}%{_bindir}/osbs %{buildroot}%{_bindir}/osbs-%{python3_version}
 ln -s  %{_bindir}/osbs-%{python3_version} %{buildroot}%{_bindir}/osbs-3
 ln -s  %{_bindir}/osbs-%{binaries_py_version} %{buildroot}%{_bindir}/osbs
 
-%if 0%{?with_check}
-%check
-LANG=en_US.utf8 py.test-%{python3_version} -vv tests
-%endif # with_check
```

Comment:
Maybe also remove the section that defines with_check
```spec
%if 0%{?fedora}
# rhel/epel has older incompatible version of pytest (no caplog)
%global with_check 1
%endif
```

Reply:
removed

Reply:
although there is another section that uses that check -
`%if 0%{?with_check}
BuildRequires:  python3-dateutil
BuildRequires:  python3-pytest
BuildRequires:  python3-flexmock
BuildRequires:  python3-six
BuildRequires:  python3-dockerfile-parse
BuildRequires:  python3-requests
BuildRequires:  python3-requests-kerberos
BuildRequires:  python3-PyYAML
BuildRequires:  python3-jsonschema
%endif # with_check`

Reply:
Ah, then maybe keep everything related to `with_check` and set it to 0 instead?

Reply:
Although that's probably going to mess with `test.sh`... why is everything always held together with duct tape? :smile: 
If it causes issues, just use what worked originally

Reply:
Works for me, as long as `test.sh` still works (I think it installs dependencies based on the specfile)

Reply:
of course it didn't work, keeping the original change :D

Reply:
could you please update the checklist?